product-mini/platforms/ios/generate_xcodeproj.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Comment:
Should also add `# Copyright (C) 2019 Intel Corporation. All rights reserved.` like sentence as the first line